virtual-desktop/src/app/window-manager/mvd-window-manager/launchbar/launchbar-icon/launchbar-icon.component.ts

```diff
@@ -23,8 +23,9 @@ export class LaunchbarIconComponent {
   @Input() launchbarItem: LaunchbarItem;
 
   @Output() iconClicked: EventEmitter<void>;
-
-  constructor(@Inject(MVDHosting.Tokens.ApplicationManagerToken) private applicationManager: MVDHosting.ApplicationManagerInterface) {
+  private applicationManager: MVDHosting.ApplicationManagerInterface;
```

Comment:
I'm curious why this change is necessary

Reply:
The ngc complained "Can't resolve all parameters for LaunchbarIconComponent..." because ApplicationManagerInterface is under a namespace.